vortex-array/src/stats/flatbuffers.rs

```diff
@@ -2,41 +2,62 @@ use flatbuffers::{FlatBufferBuilder, WIPOffset};
 use itertools::Itertools;
 use vortex_flatbuffers::WriteFlatBuffer;
 
-use crate::stats::{Stat, Statistics};
+use crate::stats::{Precision, Stat, Statistics};
 
 impl WriteFlatBuffer for &dyn Statistics {
     type Target<'t> = crate::flatbuffers::ArrayStats<'t>;
 
+    /// All statistics written must be exact
     fn write_flatbuffer<'fb>(
         &self,
         fbb: &mut FlatBufferBuilder<'fb>,
     ) -> WIPOffset<Self::Target<'fb>> {
         let trailing_zero_freq = self
             .get_as::<Vec<u64>>(Stat::TrailingZeroFreq)
-            .map(|v| v.iter().copied().collect_vec())
+            .map(|v| v.ok_exact().iter().flatten().copied().collect_vec())
```

Comment:
This line is unreadable, what's it doing?

Reply:
Renamed to inexact

Reply:
Then its just the Option<Vec<>> -> Vec flatten.